DataStructures.Tests/AATreeTests.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using DataStructures.AATree;
+
+using NUnit.Framework;
+
+namespace DataStructures.Tests
+{
+    class AATreeTests
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6367315)